airbrake/handler.py

```diff
@@ -25,7 +25,7 @@ class AirbrakeHandler(logging.Handler):
     # pylint: disable=too-many-arguments
 
     def __init__(self, airbrake=None, level=logging.ERROR, project_id=None,
-                 api_key=None, environment=None, base_url=None, hostname=None):
+                 api_key=None, **config):
```

Comment:
Yes, alot of the information that'll end up in `context` is going to be in our `**config`, but we'll have other things in the future as well, like a value for the request `timeout`, or a flag to use a non-blocking request library. I've just called it `config` since it's a bit of a grab bag, and I don't want to be changing this signature too often.

The schema for the notifications endpoint is up here: https://airbrake.io/docs/api/#error-notification-v3